packages/@sanity/desk-tool/src/components/BrokenReferences.js

```diff
@@ -22,50 +25,62 @@ function BrokenRefs(props) {
   )
 
-  const isMultiDoc = documents.length > 1
-  const documentsText = isMultiDoc ? 'other documents' : 'another document'
-  const actionDocText = isMultiDoc ? 'these other documents' : 'the other document'
-  let missingText = 'do not exist'
-  let actionText = 'must be created'
-  if (unpublished.length > 0 && nonExistent.length > 0) {
-    missingText = 'do not exist and/or are not published'
-    actionText = 'must first be created and published'
-  } else if (unpublished.length > 0) {
-    missingText = `${isMultiDoc ? 'are' : 'is'} not published`
-    actionText = 'must first be published'
-  }
-
+  const renderNonExisting = nonExistent.length > 0
+  const renderUnpublished = !renderNonExisting
   return (
-    <DefaultPane title="Broken references">
-      <p>
-        The initial value for this document references {documentsText} that {missingText}. In order
-        to create <em>this</em> document, {actionDocText} {actionText}.
-      </p>
-
-      {unpublished.length > 0 && (
-        <>
-          <h2>Unpublished documents:</h2>
-          <ReferringDocumentsList
-            documents={unpublished.map(({id, type}) => ({_id: `drafts.${id}`, _type: type}))}
-          />
-        </>
-      )}
-
-      {nonExistent.length > 0 && (
-        <>
-          <h2>Non-existent document IDs:</h2>
-          <ul>
-            {nonExistent.map(doc => (
-              <li key={doc.id}>{doc.id}</li>
-            ))}
-          </ul>
-        </>
-      )}
+    <DefaultPane title={`New ${type}`} contentMaxWidth={672} isScrollable={false}>
+      <div className={styles.brokenReferences}>
+        {renderNonExisting && (
+          <PanePopover
+            icon
+            kind="error"
+            id="missing-references"
+            title="Missing references"
+            subtitle={`The new document can only reference existing documents. ${
+              nonExistent.length === 1 ? 'A document' : 'Documents'
+            } with the following ID${nonExistent.length === 1 ? ' was' : 's were'} not found:`}
+          >
+            <ul className={styles.tagList}>
+              {nonExistent.map(doc => (
+                <li className={styles.tagItem} key={doc.id}>
+                  {doc.id}
+                </li>
+              ))}
+            </ul>
+          </PanePopover>
+        )}
+        {renderUnpublished && (
+          <PanePopover
+            icon
+            kind="warning"
+            id="unpublished-documents"
+            title="Unpublished references"
+            subtitle={`A document can only refer to published documents. Publish the following ${
+              unpublished.length === 1 ? 'draft' : 'drafts'
+            } before creating
+            a new document.`}
+          >
+            <ReferringDocumentsList
+              documents={unpublished.map(({id, type, hasDraft}) => ({
+                _id: `drafts.${id}`,
+                _type: type,
+                _hasDraft: hasDraft
```

Comment:
A bit of a nitpick, but I feel like the `_hasDraft` property here is a little dirty - given it's not actually a part of the document. Not sure what the best approach is. Maybe send an array of `drafts` (ids) along with `documents`? 
Or just unprefix the properties (`_id` => `id`) and map them back in the `ReferringDocumentsList` component?
Maybe @bjoerge has some better ideas.

Reply:
Ok! Tried to follow what was already in there, but would love some more input. I think `ReferringDocumentsList` would become more flexible if whether to show a status badge or not could be baked in. Was unsure how to determine if a document was a draft by looking at other preview components and their badges.

Reply:
I don't think is critical - we're not exposing it to other people, so we can always clean this up later.